tests/bug-5486/test.yaml

```diff
@@ -0,0 +1,11 @@
+requires:
+  min-version: 7
```

Comment:
For now, please keep the min-version to 8 to avoid having CI in a broken state until this is backported